icp/src/python/triton_distributed/icp/eventplane.py

```diff
@@ -0,0 +1,98 @@
+import uuid
+from abc import abstractmethod
+from dataclasses import dataclass
+from datetime import datetime
+from typing import List, Optional, Union
+
+from triton_distributed.icp.protos import event_pb2
+
+
+class EventTopic:
```

Comment:
Potential as we are in icp.event_plane could drop event prefix - just thought

```suggestion
class Topic:
```